.storybook/recipes/ButtonActionCalloutCard/ButtonActionCalloutCard.tsx

```diff
@@ -0,0 +1,58 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './ButtonActionCalloutCard.module.css';
+import { Card, Heading, Text } from '../../../src';
+
+export interface Props {
+  /**
+   * Slot for buttons or other calls to action
+   */
+  actions?: ReactNode;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Child node(s) that can be nested inside component
+   */
+  children?: ReactNode;
+  /**
+   * Title of the card
+   */
+  title?: string;
+}
+
+/**
+ * Card recipe with treatment to bring attention to the user. This contains a title,
+ * description, and slot for actions.
+ */
+export const ButtonActionCalloutCard = ({
+  actions,
+  className,
+  title,
+  children,
+  ...other
+}: Props) => {
+  const componentClassName = clsx(
+    styles['button-action-callout-card'],
```

Comment:
Where is `button-action-callout-card` being defined? I only see `button-action-callout-card__body-inner` in `.storybook/recipes/ButtonActionCalloutCard/ButtonActionCalloutCard.module.css`.

Reply:
@dierat This is more of how we reference the recipe (always have a base class for that). I can remove it if you think it's an issue since styles aren't applied.